test/http-server-test.js

```diff
@@ -155,6 +155,43 @@ vows.describe('http-server').addBatch({
       },
       'response Access-Control-Allow-Headers should contain X-Test': function (err, res) {
         assert.ok(res.headers['access-control-allow-headers'].split(/\s*,\s*/g).indexOf('X-Test') >= 0, 204);
+      },
+      'response Access-Control-Allow-Credentials should not be presented': function (err, res) {
+        assert.equal(res.headers['access-control-allow-credentials'], undefined);
+      }
+    }
+  },
+  'When credentials is enabled': {
+    topic: function () {
+      var server = httpServer.createServer({
+        root: root,
+        credentials: true,
+        corsHeaders: 'X-Test'
+      });
+      server.listen(8083);
```

Comment:
This port is already being used by username/password tests, so we should do 8085 here (8084 is also claimed). This is causing the tests to fail.

Reply:
👌